sql/sql_parse.cc

```diff
@@ -2572,7 +2572,7 @@ static bool lock_tables_for_backup(THD *thd)
   @return FALSE in case of success, TRUE in case of error.
 */
 
-static bool lock_binlog_for_backup(THD *thd)
+bool lock_binlog_for_backup(THD *thd)
```

Comment:
As Laurynas said, sometimes if the change is trivial, check with him to decide if a bp is really necessary.